includes/class-edd-api.php

```diff
@@ -1274,6 +1283,87 @@ public function get_discounts( $discount = null ) {
 
 
 	/**
+	 * Process Get Downloads API Request
+	 *
+	 * @access public
+	 * @since 2.5
+	 * @author Daniel J Griffiths
+	 * @global object $wpdb Used to query the database using the WordPress
+	 *   Database API
+	 * @return array $downloads Multidimensional array of the downloads
+	 */
+	public function get_downloads( $customer ) {
+		$downloads  = array();
+		$errors     = array();
+		$meta_query = array();
+
+		global $wpdb, $edd_logs;
+
+		$paged    = $this->get_paged();
+		$per_page = $this->per_page();
+		$offset   = $per_page * ( $paged - 1 );
+		$count    = 0;
+
+		if( $customer ) {
+			if( is_numeric( $customer ) ) {
+				$meta_query[] = array(
+					'key'   => '_edd_log_user_id',
+					'value' => $customer
+				);
+			} else {
+				$meta_query[] = array(
+					'key'     => '_edd_log_user_info',
+					'value'   => $customer,
+					'compare' => 'LIKE'
+				);
+			}
+		}
+
+		$query = array(
+			'log_type'       => 'file_download',
+			'paged'          => $paged,
+			'meta_query'     => $meta_query,
+			'posts_per_page' => $per_page,
+			'update_post_meta_cache' => false,
+			'update_post_term_cache' => false
+		);
+
+		$logs = $edd_logs->get_connected_logs( $query );
+
+		if ( empty( $logs ) ) {
+			$error['error'] = __( 'No downloads found!', 'edd' );
```

Comment:
Can we update this to say `No download logs found!`

The phrasing might lead someone to believe all their products are missing.